robot/impl/local_robot.go

```diff
@@ -1115,6 +1115,8 @@ func (r *localRobot) Reconfigure(ctx context.Context, newConfig *config.Config)
 		return
 	}
 
+	r.logger.CInfo(ctx, "Reconfiguring robot")
```

Comment:
if we can't tell if its the initial configuration or subsequent reconfigurations, can we use (Re)configuring?

Reply:
Yep! I've also used the word "Robot" here instead of "Smart machine"; I think we should have a larger "robot->smart machine" ticket if that's something we're interested in doing.